### DISCUSSION TITLE
Whisky & Star Citizen

### DISCUSSION THREAD

**Discussion by guruk** [Whisky-App/Whisky#747] [General]
I got several games work with whisky..... star citizen not.
while installation i got a dotfx .net framework error. than a javascript error.
anyone got that game to run?

**Comment by c01nd01r** [comment]
Star Citizen requires AVX.
https://robertsspaceindustries.com/spectrum/community/SC/forum/1/thread/rsi-launcher-update-avx-requirements

Apple Silicon chips use ARM architecture, do not support AVX instructions.
